src/site/content/en/design-system/css-utilities.md

```diff
@@ -76,6 +96,27 @@ chunks/sections/regions of content.
 <div class="region">I have consistent space now.</div>
 ```
 
+## Scrollbar
+
+Provides a scrollbar only if there is overflow content. This utility only provides the scrollbar and **not** the rule which determines which direction scrolling occurs (`overflow-x`, `overflow-y`, `overflow`), so the element that this is applied to needs to deal with that.
+
+The scrollbar colors can be changed too by setting the following **on the element** or on a parent element that has a [higher specificity](https://web.dev/learn/css/specificity/) than `:root`.
+
+<dl>
```

Comment:
We actually have a Markdown formatter which supports these elements.

I think it's:

```
Title text like a paragraph

: Definition text like a dot point
  It can extend over lines
```

Reply:
You don't need to group it with anything, the presence of a : following a normal paragraph starts the list.